packages/caliper-core/lib/caliper-flow.js

```diff
@@ -42,6 +42,65 @@ module.exports.run = async function(absConfigFile, absNetworkFile, admin, client
     let successes = 0;
     let failures = 0;
 
+    // High level flow specifications from user
+    let flowFlags = 0;
+    let performStart;
+    let performInit;
+    let performInstall;
+    let performTest;
+    let performEnd;
+    if (Config.get(Config.keys.Flow.StartOnly, false)) {
+        logger.info('Flow flag detected, will only execute start scripts');
+        performStart = true;
+        performInit = false;
+        performInstall = false;
+        performTest = false;
+        performEnd = false;
+        flowFlags++;
+    } else if (Config.get(Config.keys.Flow.InitOnly, false)) {
+        logger.info('Flow flag detected, will only perform init phase');
+        performStart = false;
+        performInit = true;
+        performInstall = false;
+        performTest = false;
+        performEnd = false;
+        flowFlags++;
+    } else if (Config.get(Config.keys.Flow.InstallOnly, false)) {
+        logger.info('Flow flag detected, will only perform smart contract install phase');
+        performStart = false;
+        performInit = false;
+        performInstall = true;
+        performTest = false;
+        performEnd = false;
+        flowFlags++;
+    } else if (Config.get(Config.keys.Flow.TestOnly, false)) {
+        logger.info('Flow flag detected, will only perform test phase');
+        performStart = false;
+        performInit = false;
+        performInstall = false;
+        performTest = true;
+        performEnd = false;
+        flowFlags++;
+    } else if (Config.get(Config.keys.Flow.EndOnly, false)) {
+        logger.info('Flow flag detected, will only execute end scripts');
+        performStart = false;
+        performInit = false;
+        performInstall = false;
+        performTest = false;
+        performEnd = true;
+        flowFlags++;
+    } else {
+        performStart = true;
+        performInit = true;
+        performInstall = true;
+        performTest = true;
+        performEnd = true;
+    }
+
+    if (flowFlags > 1) {
```

Comment:
I think will always be false since the `if-else` structure imposes precedence on the flags, and only one will be processed.

Reply:
😂  - good point!!